server/service/src/rnr_form/tests/finalise.rs

```diff
@@ -71,11 +71,6 @@ mod finalise {
             .context(mock_store_a().id, "".to_string())
             .unwrap();
 
-        let last_requisition_number = RequisitionRowRepository::new(&context.connection)
```

Comment:
This was just a test warning, I'd removed the need for this already but hadn't removed this query...